src/routers/controllers/loginController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+import db from '../../models';
+import UserService from '../../services/usersService';
+import { controllerErrorHandler } from '../../utils';
+
+const UserModel = db.user;
+
+const userServiceInstance = new UserService(UserModel);
+
+export const loginController = async ({ body: { userName, password }, params: { id } }: Request, res: Response) => {
+    try {
+        const user = await userServiceInstance.readUser(id, true);
+
+        if (!user) return res.status(404).json({ message: `User with id ${id} not found` });
+        if (user.login !== userName || user.password !== password) {
+            return res.status(403).json({ message: 'Bad username / password combination' });
```

Comment:
It should be 401. There is a big diff between 401 and 403 status codes. Please refer:
[401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) - lack of valid credentials.
[403](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403) - lack of rights.